spi/identity-hub-spi/src/main/java/org/eclipse/edc/identityhub/spi/credentials/transformer/DefaultCredentialEnvelopeTransformerRegistry.java

```diff
@@ -12,31 +12,28 @@
  *
  */
 
-package org.eclipse.edc.identityhub.processor.data;
-
-import org.eclipse.edc.identityhub.spi.processor.data.DataValidator;
-import org.eclipse.edc.identityhub.spi.processor.data.DataValidatorRegistry;
+package org.eclipse.edc.identityhub.spi.credentials.transformer;
 
 import java.util.HashMap;
 import java.util.Map;
 
-public class DataValidatorRegistryImpl implements DataValidatorRegistry {
+public class DefaultCredentialEnvelopeTransformerRegistry implements CredentialEnvelopeTransformerRegistry {
 
 
-    private final Map<String, DataValidator> validators = new HashMap<>();
+    private final Map<String, CredentialEnvelopeTransformer> validators = new HashMap<>();
 
 
-    public DataValidatorRegistryImpl() {
+    public DefaultCredentialEnvelopeTransformerRegistry() {
 
     }
 
     @Override
-    public void register(DataValidator validator) {
+    public void register(CredentialEnvelopeTransformer validator) {
```

Comment:
Why not using the same approach for registering as in the verifier registry? i.e. passing the data format and the transformer in two different arguments. 

This would allow to remove the `String dataFormat()` method from the verifier interface.

Reply:
The main reason that one transformer is binded to a specific format. 
In this case we could not register a transformer for a different data format.